locator/topology.hh

```diff
@@ -87,6 +88,10 @@ public:
 
     static std::string to_string(state);
 
+    netw::msg_addr msg_addr() const noexcept {
+        return netw::msg_addr(_endpoint, 0, _host_id);
```

Comment:
s/`0`/`current_shard()`/
Or do we completely ignore the shard part?

Reply:
While initializing to this_shard_id() makes sense, the default value is currently 0
and we currently ignore it anyhow.

Reply:
ok, no point to init it to `current_shard()` in one place if it is ignored/zeroed everywhere else